src/domains/course/services/lectureCreateService.ts

```diff
@@ -28,17 +28,10 @@ export const createLecture = async (
     formData.append('file', file);
   }
 
-  console.log('📤 강의 생성 요청:', {
-    courseId,
-    sectionId,
-    payload,
-    hasFile: !!file,
-  });
-
-  return await fetchApi<CreateLectureResponse>(
+  return await postApi<CreateLectureResponse>(
     `/api/instructor/courses/${courseId}/sections/${sectionId}/lectures`,
+    undefined,
```

Comment:
```suggestion
null
```
undefined 보단, body가 없다면 null로 보내시는거 추천드립니다